index.html

```diff
@@ -0,0 +1,1148 @@
+            ></path>
+          </svg>
+        </div>
```

Comment:
You need to make the logo a link

Reply:
We have now wrapped the <svg> element in an <a> element that links to the New York Times main/landing page.